src/main/java/com/google/maps/model/Photo.java

```diff
@@ -0,0 +1,28 @@
+package com.google.maps.model;
+
+/**
+ * <p>This describes a photo available with a Search Result.</p>
+ *
+ * <p>Please see <a href="https://developers.google.com/places/web-service/photos">Photos</a> for more details.</p>
+ */
+public class Photo {
```

Comment:
I'm wondering how best to handle this in the API. We have another end point to add that returns the raw image itself, which means we have to prevent the GSON infrastructure kicking in. However, we still want the API key and URL Signing code to all work. Paging @markmcd for input...